__test__/integration/debt_token_api/scenarios/owner_of.ts

```diff
@@ -1,18 +1,60 @@
 import { DebtTokenScenario } from "./scenarios";
 import { Orders } from "./orders";
-import { ACCOUNTS } from "../../../accounts";
-import * as singleLineString from "single-line-string";
+import { DebtTokenAPIErrors } from "src/apis/debt_token_api";
 
-export const OWNER_OF_SCENARIOS: DebtTokenScenario.OwnerOfScenario[] = [
-    {
-        description: "the creditor is the owner of the debt token",
-        orders: Orders.ALL_ORDERS,
-        ...Orders.WHO,
-    },
-    {
-        description: "the creditor transfers ownership",
-        orders: Orders.ALL_ORDERS,
-        ...Orders.WHO,
-        shouldTransferTo: ACCOUNTS[2].address,
-    },
-];
+const defaults = {
+    orderFilledByCreditorOne: Orders.CREDITOR_ONE_ORDER,
+    orderFilledByCreditorTwo: Orders.CREDITOR_TWO_ORDER,
+    tokenID: (creditorOneTokenID, creditorTwoTokenID, nonexistentTokenID, malFormedTokenID) =>
```

Comment:
why is `creditorTwoTokenID` an option if neither of the `TokenInjectable` scenario types we've created so far ever leverage it?

Reply:
`TokenInjectable` should handle all four scenarios we care about when it comes to injecting token ids:

1 - You're the owner of the token
2 - You're not the owner of the token
3 - The token id does not exist
4 - The token id is malformed

Not every test suite will leverage all four scenarios though. In our case, calling ownerOf doesn't require you to be the owner of the token, which is why we don't leverage it in our tests.